### ISSUE TITLE
`budgie-run-dialog` shows quite slowly

### ISSUE THREAD

**Issue by Bios-Marcel** [budgie-desktop/budgie-desktop#1445]
**Operating System**: Solus 3.9997
**Budgie version**: budgie-desktop v10.4-70-g15c960cc

**The issue encountered**

The `budgie-run-dialog` shows quite slowly which leads to it not being able to capture the first keystroke if i type fast. The same situation is fine for the budgie menu though.

**Steps to reproduce the issue**

Press `Alt + F2` (or whatever the keybinding is for you) and type some application name instantly after pressing `Alt + F2`.

**Comment by cunidev** [comment]
Agree. Should be part of Budgie instead of a separate, independent app and kept in RAM (just with no visibility as widget) IMO

**Comment by kareemkasem** [comment]
still an occurring issue
